README.md

```diff
@@ -52,6 +52,17 @@ To make forms inside of `(comment ...)` forms appear as top-level forms for eval
 (setq clojure-ts-toplevel-inside-comment-form t)
 ```
 
+### Fill paragraph ###
```

Comment:
I'd remove the closing `###` as we don't use them anywhere else in the file.

Reply:
Removed. For some reason `markdown-mode` does it by default this way.